ports/mimxrt/pin.h

```diff
@@ -87,7 +88,14 @@ enum {
     PIN_DRIVE_POWER_6, // R0/7
 };
 
-
+enum {
+    PIN_IRQ_NONE = 0,
+    PIN_IRQ_LOW = 1,
+    PIN_IRQ_HIGH = 2,
+    PIN_IRQ_RISING = 3,
+    PIN_IRQ_FALLING = 4,
+    PIN_IRQ_RISING_OR_FALLING = 0,
+};
```

Comment:
looks like this enum can be removed

Reply:
I have to look into it, but most probably you are right.